mscore/qml/palettes/PaletteOptionsPopup.qml

```diff
@@ -0,0 +1,97 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2019 Werner Schweer and others
```

Comment:
This indeed should be `//  Copyright (C) 2020 MuseScore BVBA and others`

Reply:
I will update it in all qml files relating to the palettes, right?

Reply:
I'd do just those 2 new files and not touch the ones you just modified

Reply:
Done!